src/brsTypes/components/RoDeviceInfo.ts

```diff
@@ -0,0 +1,687 @@
+            result.push({ name: new BrsString("VendorUSBName"), value: new BrsString("") });
+            result.push({ name: new BrsString("ScreenSize"), value: new BrsString("71") });
+
+            return new RoAssociativeArray(result);
+        },
+    });
+
+    private getFriendlyName = new Callable("getFriendlyName", {
+        signature: {
+            args: [],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            return new BrsString("Roku 5000X, Serial Number: 12121212121212");
+        },
+    });
+
+    private getOSVersion = new Callable("getOSVersion", {
+        signature: {
+            args: [],
+            returns: ValueKind.Object,
+        },
+        impl: (_interpreter) => {
+            let result = new Array<AAMember>();
+            result.push({ name: new BrsString("name"), value: new BrsString("9") });
+            result.push({ name: new BrsString("minor"), value: new BrsString("2") });
+            result.push({ name: new BrsString("revision"), value: new BrsString("6") });
+            result.push({ name: new BrsString("build"), value: new BrsString("4127") });
+
+            return new RoAssociativeArray(result);
+        },
+    });
+
+    private getVersion = new Callable("getVersion", {
+        signature: {
+            args: [],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            return new BrsString("034.08E01185A");
+        },
+    });
+
+    private getRIDA = new Callable("getRIDA", {
+        signature: {
+            args: [],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            return new BrsString("123e4567-e89b-12d3-a456-426655440000");
+        },
+    });
+
+    private isRIDADisabled = new Callable("isRIDADisabled", {
+        signature: {
+            args: [],
+            returns: ValueKind.Boolean,
+        },
+        impl: (_interpreter) => {
+            return BrsBoolean.True;
+        },
+    });
+
+    private getChannelClientId = new Callable("getChannelClientId", {
+        signature: {
+            args: [],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            return new BrsString("123e4567-e89b-12d3-a456-426655440000");
+        },
+    });
+
+    private getUserCountryCode = new Callable("getUserCountryCode", {
+        signature: {
+            args: [],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            let countryCode = process.env.LOCALE;
+            return countryCode ? new BrsString(countryCode) : BrsInvalid.Instance;
+        },
+    });
+
+    private getRandomUUID = new Callable("getRandomUUID", {
+        signature: {
+            args: [],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            return new BrsString("123e4567-e89b-12d3-a456-426655440000");
```

Comment:
Instead of returning a static `uuid` here, let's generate a random one the same way that Brightscript does. We can use the [uuid npm package](https://www.npmjs.com/package/uuid).